### PR TITLE
[Tests] Extract common folder from test configuration

### PR DESCRIPTION
**Pull request by dkijania**
For future development there is a must to extract common folder from test cfg and use it as dev module

### CODE REVIEW

**Comment by CodeSandwich** [comment]
For me this PR in unreviewable. Almost 5000 added lines over 18 files is way more than I can digest in a single hit. I can be the one to click "merge", but I'm not able to actually review the changes. This PR is clearly much more than 1 added feature, so it can and should be split.

**Comment by dkijania** [comment]
Ok, I'll  try to split PR into smaller pieces

**Comment by dkijania** [comment]
I think you can take a look right know. I've added a little too many dependencies that are not needed at this moment, but once I prepare another PR, just after this they will be used

File: jormungandr-integration-tests/src/common/jcli_wrapper/certificate/wrapper.rs [hunk 1]
```diff
@@ -10,6 +10,9 @@ use chain_impl_mockchain::{
     certificate::PoolId, key::EitherEd25519SecretKey,
     testing::builders::cert_builder::build_stake_pool_retirement_cert,
 };
+
+extern crate jcli;
```
Comment (CodeSandwich) [inline jormungandr-integration-tests/src/common/jcli_wrapper/certificate/wrapper.rs]: This is useless I think

**Pull request merged**